other/huffmancoding/huffmancoding.go

```diff
@@ -0,0 +1,123 @@
+// huffman.go
```

Comment:
I believe having a separate directory for `compression` makes sense. I believe you should move this package inside a new `compression` package by creating a new directory `compression` in the root of this repository, and call that package `compression` (though feel free to create a new subpackage inside the `compression` package called `huffman` if that's easier - but in Go we try to keep similar things as close together as possible without the possibility of name clashes)